### DISCUSSION TITLE
Snapshot class for Helm Charts

### DISCUSSION THREAD

**Discussion by waza-ari** [ceph/ceph-csi#4516] [General]
Dear team,

I'm using the helm charts to deploy both the RBD and CSI driver, which works wonderfully. Wonder though if there's a reason that both to not contain options to allow snapshots, too?

Thanks

**Comment by nixpanic** [comment]
I don't think there is a technical reason that this is not included. It would surely be nice to have! PRs that add the VolumeSnapshotClass are very welcome!
